src/test/dashboard/widgets/Number.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { shallow } from 'enzyme'
+
+import Number from '../../../components/dashboard/widgets/Number'
+import widgetData from '../../../reducers/widgets/dummy-data/dashboard-tools/number.js'
+
+describe( 'Number widget component', () => {
+  const statistic = '200'
+  const goal = '200'
```

Comment:
Instead of defining the variables explicitly here, why not use widget data that was imported on line 6? ( Jason said this...not me)